src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+/* #region stars */
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
+    background: no-repeat url(/images/star.svg) center;
+
+  }
+
+  &__star:not(:last-child) {
```

Comment:
apply code from previous task (about stars) here. To make your code flexible (for different variant of stars)